src/status_im/config.cljs

```diff
@@ -160,5 +160,5 @@
 
 (def default-kdf-iterations 3200)
 
-(def community-accounts-selection-enabled? false)
+(def community-accounts-selection-enabled? true)
```

Comment:
@ilmotta - maybe it's nicer if we have a feature-flags file? 🤔

Reply:
e.g then it reads like 
`(if feature-flag/community-accounts-selection-enabled? ...`

Reply:
@J-Son89, the way other flags are set in that namespace is to use react-native-config. We didn't, but if we originally defined the flag as:

```clojure
(def community-accounts-selection-enabled? (enabled? (get-config :COMMUNITY_ACCOUNTS_SELECTION_ENABLED? "1")))
```

In theory it should be possible to generate builds directly from Jenkins because we can fire a job with that env var set to `1`, so without the need to create a WIP PR like what we did here. It's an idea I never explored, but would be quite useful due to its flexibility.

@siddarthkay, have you done such a thing in this repo?

Reply:
> e.g then it reads like `(if feature-flag/community-accounts-selection-enabled? ...`

Good idea! I would also prefer a separate feature flags file.

Reply:
> have you done such a thing in this repo

I don't think I have, but sounds cool and worth exploring :)

Reply:
Yeah I think we need to start doing this :)

Reply:
Cc @shivekkhurana @smohamedjavid

Reply:
@ilmotta - You are right. We can enable the account selection for the community for specific builds such as Dev and PR builds. This will help us to reduce the effort to keep this PR updated all the time when `develop` is updated.

We need to update the `def` to read the env.

```clojure
;; Setting default as zero as we need to enable only on dev and PR builds for now
(def community-accounts-selection-enabled? (enabled? (get-config :COMMUNITY_ACCOUNTS_SELECTION_ENABLED "0"))) 
```

Additionally, we need to update (add `COMMUNITY_ACCOUNTS_SELECTION_ENABLED=1`) the `.env` files of dev and PR at the root of the repository.

```
.env         ;;dev 
.env.e2e     ;;E2E
.env.jenkins ;;PR
.env.nightly ;;Nightly
.env.release ;;Release
```

A good example of this scenario is the quo preview screens (`ENABLE_QUO_PREVIEW`) where we show only for dev and PR builds

Reply:
@J-Son89, another option would be to revamp how we allow users (internal users) to enable feature flags in the UI. Nothing fancy, but more explicit and more automated on the CLJ layer. The solution would be focused initially in non-prod builds.

The objectives could be:

1. Allow designers and QAs to explore the app as its being built, and detached from environments (PRs, dev, local, and so on).
2. No need for developers to come up with clever solutions, like hold for 2s to press.
3. It should dead easy to add a feature flag and make it available in the UI.

These ideas would probably spark some refactors in the `status-im.config` namespace too.

Build generation while easily allowing overriding the default (per env) feature flags seems to be more work and more difficult to use, although I still like that.

But let's keep talking about this!

Reply:
Yep I agree, this sounds great to me 👌

Reply:
> But for this idea to work, it would be ideal to show all the available feature flags in the web form before firing a job, so that the person can decide which flags to override.

Currently, Nix switches the env file depending on the build type. The secrets (like Wallet secrets) are injected from Jenkins directly based on the build type. To provide UI on Jenkins for overriding where we can build with parameters would require a bit of effort.

On the other hand, I liked the idea of having a screen to toggle features that are under development or experimental. This requires minimum effort comparatively and a cleaner solution to provide what the user (dev, QA, and design) wants from the app. Clean and neat 👍

Reply:
> Currently, Nix switches the env file depending on the build type. The secrets (like Wallet secrets) are injected from Jenkins directly based on the build type. To provide UI on Jenkins for overriding where we can build with parameters would require a bit of effort.

Thanks for the info!

Reply:
@ilmotta, @smohamedjavid can you check https://github.com/status-im/status-mobile/pull/18602